content/community/resources/case-studies/kuksa_breaksdown_automotive_silos.md

```diff
@@ -0,0 +1,88 @@
+
+Eclipse Kuksa provides an open, accessible platform that allows all players in the automotive industry to take a common approach to vehicle-tocloud scenarios and reduce the costs associated with repetitive development efforts.
+
+## Unifying Frameworks and Technologies
+
+Eclipse Kuksa unifies vehicle, Internet of Things (IoT), cloud, and security technologies across the complete tooling stack for the connected vehicle domain to enable a standardized approach to Vehicle-To-Cloud (V2C) scenarios across all vehicles (Figure 1). It provides. 
+
+* **An in-vehicle platform.** The in-vehicle platform consists of a container-based app runtime and supporting facilities that sit on top of a Linux-based operating system, such as Automotive Grade Linux (AGL), Apertis, or Debian. 
+
+The platform provides a controlled environment to execute new vehicle functionality as well as access to vehicle data. Data access is provided in a secure and standardized way using open source implementations of the World Wide Web Consortium (W3C) Vehicle Information Service Specification (VISS) standard and the SENSOR Interface Specification (SENSORIS) standard for collecting vehicle and sensor data. Compliance with these standards makes it easier for vehicle manufacturers of all sizes to meet European Union regulations for accessing in-vehicle data. 
+
+* **A cloud backend.** The cloud backend integrates several projects from the [Eclipse IoT Working Group](https://iot.eclipse.org/), including [Eclipse Hono](https://www.eclipse.org/hono/), [Eclipse hawkBit](https://www.eclipse.org/hawkbit/), and in future releases, [Eclipse Ditto](https://www.eclipse.org/ditto/), to provide a common, Kubernetes-based deployment for an automotive IoT cloud backend. It also includes an app store that offers a user-focused management interface and automated deployment of in-vehicle apps using Eclipse hawkBit as the underlying technology. 
+
+* **An IDE.** The IDE, which is based on [Eclipse Che](https://www.eclipse.org/che/) and Microsoft Visual Studio Code, automates in-vehicle app development. The IDE interacts with the app store to simplify app deployments. 
+
+Future Kuksa releases will include [Eclipse Theia](https://theia-ide.org/) extensions. These extensions comply with Eclipse Che 7 and support the Kuksa in-vehicle application build processes, application registrations, and transmissions to the Eclipse Kuksa Appstore and the Eclipse Kuksa Cloud.
+
+Like all research projects at the Eclipse Foundation, Kuksa fosters open collaboration across academic, business, and public sector interests. Along with academic institutions from Germany, the Netherlands, and Finland, a number of corporations have joined the project.
+
+![Figure 1: The Eclipse Kuksa Ecosystem (Third-party trademarks are the property of their respective owners.)](/community/resources/case-studies/images/kuksa-figure-1.png)
```

Comment:
Try and add the classes "img img-responsive margin-auto" to this image. Otherwise it mucks up the page and doesn't look good at mobile. We have a figure shortcode that may be useful for this!